sympy/solvers/inequalities.py

```diff
@@ -378,7 +379,7 @@ def reduce_abs_inequalities(exprs, gen):
         for expr, rel in exprs ])
 
 
-def solve_univariate_inequality(expr, gen, relational=True):
+def solve_univariate_inequality(expr, gen, domain=S.Reals, relational=True):
```

Comment:
This new kwarg should be added after relational to preserve the order and backwards compatibility.